DNN Platform/Library/Services/FileSystem/FileManager.cs

```diff
@@ -899,7 +899,7 @@ public virtual IFileInfo GetFile(int fileID, bool retrieveUnpublishedFiles)
             var file = DataCache.GetCache(strCacheKey);
             if (file == null)
             {
-                file = CBO.Instance.FillObject<FileInfo>(DataProvider.Instance().GetFileById(fileID, retrieveUnpublishedFiles));
```

Comment:
its not about unit tests for CBO, but other unit tests depend on CBO, and they will set mockup CBO when needed, like used in file manager tests.